sdkproject/Assets/Mapbox/Unity/Map/AbstractMap.cs

```diff
@@ -365,9 +365,14 @@ public virtual void UpdateMap(Vector2d latLon, float zoom)
 		/// Resets the map.
 		/// Use this method to reset the map to and reset all parameters.
 		/// </summary>
+		[ContextMenu("ResetMap")]
```

Comment:
Is this context menu for testing or do we want to commit it?

Reply:
I would love to have it in actually. My map times out quite a lot of time because of slow internet and with that button I can just reload everything easily so it's pretty useful.